common/client.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// Client holds configuration data associated with the HTTP(s) session
+type Client struct {
+	HTTPClient http.Client
+	// TODO(tho) API tokens and client credentials
```

Comment:
Create a github issue for the // TO DO  and refer it here!

Reply:
```suggestion
```